db/schema.rb

```diff
@@ -121,6 +121,30 @@
     t.index ["eventable_id", "eventable_type"], name: "index_events_on_eventable_id_and_eventable_type"
   end
 
+  create_table "framework_questions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "framework_id", null: false
+    t.string "key", null: false
+    t.string "section", null: false
+    t.boolean "required", default: false, null: false
+    t.string "question_type", null: false
+    t.string "options", default: [], array: true
+    t.string "dependent_value"
+    t.boolean "followup_comment", default: false, null: false
+    t.string "followup_comment_options", default: [], array: true
+    t.uuid "parent_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["framework_id"], name: "index_framework_questions_on_framework_id"
+    t.index ["parent_id"], name: "index_framework_questions_on_parent_id"
+  end
+
+  create_table "frameworks", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "name", null: false
+    t.decimal "version", null: false
```

Comment:
Decimal is a bit of an odd choice for a version number - this might get some freaky rounding issues, and also precludes the use of semver. Would you consider a string type instead?

Reply:
yep, I agree - string is preferable to decimal